WildShape-Sheets-API/Controllers/UserController.cs

```diff
@@ -11,9 +11,15 @@ namespace WildShape_Sheets_API.Controllers {
     public class UserController : Controller {
 
         private readonly UserService userService;
+        private readonly EmailService emailService;
+        private readonly HashService hashService;
```

Comment:
These should have _ in front  and the constructor params should not have it. These two are backwards

Reply:
this may have been originally my doing, which i was fixing else where